DEPLOYMENT.md

```
@@ -0,0 +1,236 @@
+npx vercel
+
+# For production
+npx vercel --prod
+```
+
+Or use GitHub integration:
+1. Push to GitHub
+2. Connect GitHub to Vercel dashboard
+3. Auto-deploys on every push to main
+
+**Your URL:** https://slerf-app.vercel.app
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)